CoverletReproductions.Test/AwaitForeachReproductionFixture2.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
```

Comment:
This is the only new file. Repro for https://github.com/coverlet-coverage/coverlet/issues/1210